agent/php_nrini.c

```diff
@@ -255,6 +255,28 @@ static void check_ini_int_min_max(int* var, int minval, int maxval) {
   }
 }
 
+/*
+ * @brief error handling for strtol() functionality
+ *
+ * @param val_p   pointer to store the parsed value
+ * @param str     string to parse
+ * @param base    base to be used for number conversion
+ * @return nr_status_t NR_SUCCESS || NR_FAILURE
+ */
+static nr_status_t nr_strtol(int* val_p, const char* str, int base) {
+  int val;
+  char* endptr_p;
+  errno = 0;
+
+  val = (int)strtol(str, &endptr_p, base);
```

Comment:
`strtol` would suggest that the function will return `long` not `int`
```suggestion
static nr_status_t nr_strtol(long* val_p, const char* str, int base) {
  long val;
  char* endptr_p;
  errno = 0;

  val = strtol(str, &endptr_p, base);
```
alternatively change the name and the C library function which may be a better option seeing how `nr_strtol` is being used in the code.
```suggestion
static nr_status_t nr_strtoi(int* val_p, const char* str, int base) {
  int val;
  char* endptr_p;
  errno = 0;

  val = strtoimax(str, &endptr_p, base);
```